src/lib/database/entities/GiveawayEntity.ts

```diff
@@ -187,8 +187,8 @@ export class GiveawayEntity extends BaseEntity {
 		const t = await guild.fetchT();
 		if (state === States.Finished) {
 			this.#winners = await this.pickWinners();
+			this.#finished = true;
```

Comment:
Will the fact that it gets shifted up a line matter?

Reply:
No, it's more preference to do 2 sets together.